model/src/main/java/com/epam/repair/model/Client.java

```diff
@@ -30,21 +30,21 @@
      */
     @ManyToOne
     @JoinColumn (name="first_name_id")
-    private FirstName firstName;
+    private FirstNameDTO firstName;
```

Comment:
Here can't be `dto` object because it's DB view

Reply:
Comment accepted. Fixed in the branch mapstruct-model.